airbyte-integrations/connectors/source-hubspot/unit_tests/test_streams.py

```diff
@@ -317,12 +429,76 @@ def test_crm_search_streams_with_no_associations(sync_mode, requests_mock, fake_
     stream._sync_mode = sync_mode
     requests_mock.register_uri("POST", endpoint_path, responses)
     requests_mock.register_uri("GET", properties_path, properties_response)
+    records = run_read(stream)
+    assert records
     if sync_mode == SyncMode.incremental:
-        records, state = read_incremental(stream, stream_state=stream_state.stream.stream_state.__dict__)
-        assert state
-    else:
-        records = read_full_refresh(stream)
+        state = stream.cursor.state
+        assert state == {"updatedAt": "2022-02-25T16:43:11.000000Z"}
+
+
+@freezegun.freeze_time("2022-02-25T17:00:00Z")
+def test_crm_search_streams_requests_contain_custom_properties(requests_mock, fake_properties_list, config):
```

Comment:
I'm fine adding this test too but wouldn't this be covered if we were to match the HTTP body for the POST request in `test_leads_incremental`?

Reply:
so I deleted `test_leads_incremental` now because that test was actually an accidental duplicate. So this is actually the intended test I wanted and I deleted `test_leads_incremental`